qcut/docs/issues/video-canva-play-fix/consolev8.md

```
@@ -0,0 +1,680 @@
+preview-panel.tsx:930 step 12a: rendering video element Object
+blob:app://./2392bd8a-5e84-4c9a-bcb9-d439ab3cd4c1:1  Failed to load resource: net::ERR_FILE_NOT_FOUND
+preview-panel.tsx:785 step 12: rendering element Object
+preview-panel.tsx:930 step 12a: rendering video element Object
+preview-panel.tsx:785 step 12: rendering element Object
+preview-panel.tsx:930 step 12a: rendering video element Object
+index.html:68 🔧 Suppressed blob network error for: VIDEO
+preview-panel.tsx:785 step 12: rendering element Object
+preview-panel.tsx:930 step 12a: rendering video element Object
+preview-panel.tsx:785 step 12: rendering element Object
+preview-panel.tsx:930 step 12a: rendering video element Object
+html2canvas.esm.js:5347 Canvas2D: Multiple readback operations using getImageData are faster with the willReadFrequently attribute set to true. See: https://html.spec.whatwg.org/multipage/canvas.html#concept-canvas-will-read-frequently
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)